lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -2927,6 +2929,8 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
           // apparently asymmetric
           if (coordTfMode != "asymmetric" && coordTfMode != "align_corners")
             modeStr = (modeStr + "_") + coordTfMode;
+          if (nearest_mode != "round_prefer_floor" && nearest_mode != "")
```

Comment:
This should still be `floor`, since that is default pytorch behavior.

Reply:
change `binder.customOpNameStringAttr(nearest_mode, "nearest_mode", ""))` from above to 
`binder.customOpNameStringAttr(nearest_mode, "nearest_mode", "round_prefer_floor"))`